notebooks/demo.ipynb

```diff
@@ -0,0 +1,75 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from birdy import WPSClient\n",
+    "import requests\n",
+    "import json\n",
+    "from pkg_resources import resource_filename\n",
+    "\n",
+    "# Setup\n",
+    "url = 'http://localhost:5003/wps'\n",
+    "sandpiper = WPSClient(url=url)\n",
+    "\n",
+    "# Run process\n",
+    "output = sandpiper.resolve_rules(\n",
+    "    csv=resource_filename(\"tests\", \"data/rules_small.csv\")\n",
+    ")"
```

Comment:
I still have the same error even after the storage is mounted:
```
Traceback (most recent call last):
  File "/home/sangwonl/Desktop/sandpiper/venv/lib/python3.7/site-packages/pywps/app/Process.py", line 249, in _run_process
    self.handler(wps_request, wps_response)  # the user must update the wps_response.
  File "/home/sangwonl/Desktop/sandpiper/sandpiper/processes/wps_resolve_rules.py", line 119, in _handler
    rules, date_range, region, ensemble, connection_string, log_level
  File "/home/sangwonl/Desktop/sandpiper/venv/lib/python3.7/site-packages/p2a_impacts/resolver.py", line 78, in resolve_rules
    if var is not None:
UnboundLocalError: local variable 'var' referenced before assignment
```
 I wonder if there is anything wrong with my installation process.
```
pip install -r requirements -i https://pypi.pacificclimate.org/simple/
pip install -e .[dev]
sudo mount -t cifs -o username=sangwonl,uid=1000,gid=50 //pcic-storage.pcic.uvic.ca/storage /storage
```

Reply:
I think I know what the issue is. Working through it.